config/checkstyle/checkstyle.xml

```diff
@@ -34,7 +34,7 @@
   </module>
   <module name="SuppressWarningsFilter" />
   <module name="LineLength">
-    <property name="max" value="120"/>
+    <property name="max" value="180"/>
```

Comment:
I'll make a note on this for other reviewers' benefit.

This is a change I made as TL. It's low-priority, doesn't affect the application logic and is within the domain of TLs, not devs.

Standard practice would be to have a separate ticket for this, so please keep in mind this is a bad practice I am adopting here and DON'T do it yourselves. I packed it in this ticket to see how reception of this change would have been.